lib/install/phases/bootstrap.go

```diff
@@ -203,6 +204,11 @@ func (p *bootstrapExecutor) configureSystemDirectories() error {
 		filepath.Join(stateDir, "secrets"),
 		filepath.Join(stateDir, "backup"),
 	}
+	p.Info(spew.Sdump(p.Application))
+	if p.Application.Manifest.Extensions != nil && p.Application.Manifest.Extensions.Monitoring != nil && !p.Application.Manifest.Extensions.Monitoring.Disabled {
```

Comment:
Does it need to be conditionally enabled? What happens if we always create/chown this directory?

Reply:
Just you know, to make things not hardcoded. Users could ask why do I have empty monitoring directories on all servers even if I disabled monitoring extension.